src/new_hope/week3.clj

```diff
@@ -0,0 +1,148 @@
+(ns new-hope.week3
+  (:require [midje.sweet :refer :all])
+  (:require [clojure.test :refer :all])
+  (:require [clojure.set :refer :all])
+  )
+
+;Day 1
+(defn is-binary-tree
+  "determines whether the argument sequence is a binary tree or not"
+  [x]
+  (if (not (coll? x))
```

Comment:
I think you need `sequential?` here, not `coll?`
https://www.brainonfire.net/files/seqs-and-colls/main.html